app/assets/templates/header.html

```diff
@@ -11,7 +11,7 @@
           </button>
           <ul class="dropdown-menu list-inline" role="menu">
             <li class="dropdown-list" ng-repeat='category in categories'>
-              <a ng-href="#{{category.link}}" class="dropdown-category">{{ category.name }}</a>
+              <a ng-href="#/categories/{{ category.link }}" class="dropdown-category">{{ category.name }}</a>
```

Comment:
do you still use the link? I thought you changed to simplified_name? so your url would look like categories/{{category.simplified_name}}